class AuthenticationError(Exception):
    """Exception for failed authentication"""
    pass